src/lib/block_if.c

```diff
@@ -372,7 +389,7 @@ blockif_proc(struct blockif_ctxt *bc, struct blockif_elem *be, uint8_t *buf)
```

Comment:
would just prefer to keep this codebase clean without the "bhyve removal" comments -- its pretty obvious from the perror that alternative implementations could exist.